jena-arq/src/main/java/org/apache/jena/sparql/engine/http/QueryExceptionHTTP.java

```diff
@@ -63,11 +66,21 @@ public QueryExceptionHTTP(int responseCode)
     public int getResponseCode() { return responseCode ; }
     
     
-    /** The messge for the reason for this exception
+    /** The message for the reason for this exception
      * @return message
      */  
     public String getResponseMessage() { return responseMessage ; }
 
+    /** The response for this exception
+     * @return response
```

Comment:
Note that this can be null if the original exception was not from the HTTP protocol, but from e.g. an IOException why doing an HTTP operation.

"The response for this exception if available from HTTP"